datasource/amazon/ami/acceptance/acceptance.go

```diff
@@ -0,0 +1,107 @@
+package acceptance
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/hashicorp/packer-plugin-sdk/acctest"
+	"github.com/hashicorp/packer-plugin-sdk/acctest/testutils"
+)
+
+// TODO: this should be remove once it's added to the packer-plugin-sdk
+
+// DatasourceTestCase is a single set of tests to run for a datasource.
+// A DatasourceTestCase should generally map 1:1 to each test method for your
+// acceptance tests.
+type DatasourceTestCase struct {
```

Comment:
This is here because currently, this code depends on two not merged packer-plugin-sdk PR that contains different code. 
https://github.com/hashicorp/packer-plugin-sdk/pull/15/files and https://github.com/hashicorp/packer-plugin-sdk/pull/11